behind-the-veil-siteroot/imports/ui/components/protectedRoute/LoggedInOnlyRoute.jsx

```diff
@@ -23,7 +23,7 @@ export const LoggedInOnlyRoute = ({children}) => {
     // todo: edit this logic once login is done
     const user = getUserAuth();
 
-    if (user === false || !user) {
+    if (false) {
```

Comment:
just wondering, is this supposed to never run?

Reply:
oh yeah, the correct code should be the previous code. I changed it to false so that I can access the messenger page without logging in. I changed back to old code.